shared/js/textChatView.jsx

```diff
@@ -79,6 +80,21 @@ loop.shared.views.chat = (function(mozL10n) {
         );
       }
 
+      if (this.props.contentType === CHAT_CONTENT_TYPES.NOTIFICATION) {
+
+        var thumbnail = "shared/img/leave_notification.svg";
```

Comment:
nit: I think we can drop the intermediate variable now this has been simplified.